### PR TITLE
Vd 8

### PR DESCRIPTION
**Pull request by wsmoyer**

### CODE REVIEW

**Review by ahackit — COMMENTED** [review]
Were you able to get some Test Layers created with that data? If so, I'd say we could probably include some example Layer creation in App.vue for the time being and pass those layers.

**Comment by wsmoyer** [comment]
> Were you able to get some Test Layers created with that data? If so, I'd say we could probably include some example Layer creation in App.vue for the time being and pass those layers.

ok. i initially started to create an example but then thought maybe that should be a separate task. What do you think?

File: App.vue [hunk 1]
```diff
@@ -25,6 +25,15 @@
...
     }
+
+
+    // urls for sample data. feel free to add more from this repo
+//     const DATA_URL = {
+//     ACCIDENTS: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/examples/highway/accidents.csv',
+//     ROADS: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/examples/highway/roads.json'
+// };
+
+
```
Comment (TyrelDenison) [inline App.vue]: 💩

**Comment by ahackit** [comment]
> > Were you able to get some Test Layers created with that data? If so, I'd say we could probably include some example Layer creation in App.vue for the time being and pass those layers.
> 
> ok. i initially started to create an example but then thought maybe that should be a separate task. What do you think?

Ya - that could be a separate task to set up test data. In that case - I'd take out test data links and we can include those in separate file/directory

**Review by ahackit — APPROVED** [review]
🍰  if you can switch this branch to point to development. 

Looks good.

**Pull request merged**